components/overflow-group/test/overflow-group.visual-diff.js

```diff
@@ -84,32 +87,30 @@ describe('d2l-overflow-group', () => {
 			name: 'opener-type-subtle-icon'
 		}
 	];
-	// const itemTypeConversion = [
-	// 	{
-	// 		name: 'all-item-types',
-	// 		action: async(selector) => {
-	// 			const overflowMenu = await getShadowElem(selector, '.d2l-overflow-dropdown');
-	// 			await overflowMenu.click();
-	// 		}
-	// 	}];
+	const itemTypeConversion = [
+		{
+			name: 'all-item-types',
+			action: async(selector) => {
+				const overflowMenu = await getShadowElem(selector, '.d2l-overflow-dropdown');
+				await overflowMenu.click();
+			}
+		}];
 	[
 		...hiddenButtonTests,
-		// ...itemTypeConversion,
+		...itemTypeConversion,
 		...minMaxTests,
 		...autoShow,
 		...iconType,
 	].forEach((test) => {
 		it(test.name, async function() {
 			const selector = `#${test.name}`;
 			const containerSelector = `#${test.name}-container`;
-			await page.evaluate(() => {
-				return new Promise(resolve => setTimeout(resolve, 0));
-			});
+			await page.$eval(containerSelector || selector, (elem) => elem.scrollIntoView());
```

Comment:
Because we are no longer capturing beyond the viewport (i.e. no longer resizing the page randomly for every test), we need to make sure we've scrolled the test into view before we take the screenshot.

Reply:
Alternately, I think we used to just set the viewport to be super tall before this `captureBeyondViewport` was a thing.

Reply:
Ah, that does sounds familiar!  Confirmed that also works, do you have thoughts on which is better?  Seems like the large viewport is a _tad_ faster for the test that needs to scroll, but this way means you don't need to increase the viewport if you add tests.  The mouse placement issue is only an issue in this case, not the large viewport case (cause it doesn't scroll to a specific place and move the mouse with it)

Reply:
I don't have feelings either way, no.

Reply:
Gonna leave it like this for now since that's the way I tested it, and we'll see if it's stable or still has issues